types/vue.d.ts

```diff
@@ -80,10 +80,28 @@ declare namespace NuxtVueI18n {
  * Extends types in vue
  */
 declare module "vue/types/vue" {
+  interface NuxtI18nSeo {
+    htmlAttrs: {
+      lang: string
+    },
+    link: [{
+      hid: string,
+      rel: string,
+      href: string,
+      hreflang: string
+    }],
+    meta: [{
+      hid: string,
+      name: string,
+      property: string,
+      content: string
+    }]
+  }
   interface Vue {
     localePath(route: RawLocation, locale?: string): string;
     switchLocalePath(locale: string): string;
     getRouteBaseName(route: RawLocation): string;
+    $nuxtI18nSeo: () => NuxtI18nSeo
```

Comment:
Is this syntax of defining a function equivalent to `$nuxtI18nSeo(): NuxtI18nSeo`? If so then maybe for consistency it could be using other syntax?

Reply:
Also missing semicolon if we are talking consistency :)

Reply:
yes, although at previous declaration from this file its no semicolon and that syntax, its already not consistent :) but changed here to be consistent within this interface